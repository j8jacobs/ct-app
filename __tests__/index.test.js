import React from "react";
import renderer from "react-test-renderer";

import App from "../app/index";

describe("<App />", () => {
  it("renders", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree.children.length).toBe(2);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
