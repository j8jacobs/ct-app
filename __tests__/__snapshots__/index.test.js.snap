// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 12,
        "paddingVertical": 24,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 32,
          "fontWeight": "bold",
          "marginBottom": 8,
        }
      }
    >
      Wallet
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#007FFF",
          "borderRadius": 40,
          "height": 40,
          "justifyContent": "center",
          "opacity": 1,
          "width": 40,
        }
      }
    >
      <Text
        style={
          {
            "color": "white",
            "fontSize": 24,
          }
        }
      >
        +
      </Text>
    </View>
  </View>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</View>
`;
